src/dev/jest/junit_reporter.js

```diff
@@ -70,8 +70,9 @@ export default class JestJUnitReporter {
       { skipNullAttributes: true }
     );
 
-    const msToIso = (ms) => (ms ? new Date(ms).toISOString().slice(0, -5) : undefined);
-    const msToSec = (ms) => (ms ? (ms / 1000).toFixed(3) : undefined);
+    const isNumeric = (val) => !isNaN(parseFloat(val)) && isFinite(val);
```

Comment:
Do we need the `!isNaN` check if `isFinite` also return false if the number is `NaN` or `undefined`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite

Reply:
It is to weed out `null`, `Infinity`, and `-Infinity`.

Reply:
btw, nice to hear from you ❤️

Reply:
> It is to weed out `null`, `Infinity`, and `-Infinity`.

Ah, got it. So I think the equivalent would be `val !== null && isFinite(val)` then right? Not a big difference but we could avoid the `parseFloat`?